### DISCUSSION TITLE
I can't get the coins out of my wallet

### DISCUSSION THREAD

**Discussion by DMS-1981** [Chia-Network/chia-blockchain#5177] [Support]
Hello I have a problem the disk was broken, the chia software was installed
i entered the mnemonic key, but once i logged in i couldn't find my coins in the app.

Wait for sync and nothing

How can they be recovered?

Thank you

**Comment by lvpcguru** [comment]
are they showing the same keys and fingerprint as you had before?

**Comment by BrandtH22** [comment]
@DMS-1981 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
